source/style.rst

```diff
@@ -21,30 +24,35 @@ Code
 ====
 Our code follows `PEP-8`_ guidelines. Please look there for any code style
 matters. Since this is a very widely used standard, Google is also very helpful
-if you have any questions.
+if you have any questions. To standardize our convention for strings, please
+use single quotes except for when the string itself contains single quotes.
 
 Docstrings
 ==========
 Please write docstrings for all modules, classes, and functions/methods. In
 general, our docstrings should mostly follow `PEP-257`_ with the exception
 of an allowance for 2-line docstrings.
 
-General Rules:
---------------
+General Rules
+-------------
 * Always use ``"""triple double quotes"""`` for docstrings.
 * End all summaries/descriptions with periods.
 * Built-in types (like str and int) should not be quoted or backticked.
 * Numeric values should be left unbackticked (e.g. 1).
 * Strings should be quoted (e.g. 'foo').
-* Use the ``:keyword:`` directive as appropriate (e.g. \:keyword:\`True\`).
+* Built-in constants and keywords can be emphasized with backticks. An
```

Comment:
Huh, didn't realize it was that easy.

Reply:
It is if we enable the intersphinx extension. I'm adding it to pcdsdevices in an upcoming PR. It also let's us reference ophyd functions, which I think is very helpful.

Reply:
Ooh, that sounds nice 👍